README.md

```diff
@@ -105,6 +105,19 @@ cd driver
 cargo test --lib
 ```
 
+## Running with linear optimization solver
+
+For this to work, you will need to have read access to the [solver's gitlab repo](https://gitlab.gnosisdev.com/dfusion/batchauctions). You will also need to have SSH Key authentication for your gitlab account enabled (see [tutorial](https://docs.gitlab.com/ee/ssh/))
+
+In your top-level git folder, create a `.ssh/` folder and move or copy the private and public ssh key that are registered with your gitlab account in there.
```

Comment:
Might wanna mention that their names have to be id_rsa and id_rsa.pub

Reply:
I think that's how they are always called, but sure.